src/Microsoft.OpenApi.Tool/OpenApiService.cs

```diff
@@ -52,43 +60,31 @@ static class OpenApiService
                     errorReport.AppendLine(error.ToString());
                 }
 
-                throw new ArgumentException(String.Join(Environment.NewLine, context.Errors.Select(e => e.Message).ToArray()));
+                throw new ArgumentException(string.Join(Environment.NewLine, context.Errors.Select(e => e.Message).ToArray()));
             }
 
-            using (var outputStream = output?.Create())
+            if (output.Exists)
```

Comment:
I think this check could better come immediately after the null checks at the top; we ideally want to fail fast.

Reply:
Good point.